showcase/angular.json

```diff
@@ -59,8 +59,8 @@
               "src/styles/styles.pcss"
             ],
             "scripts": [],
-            "deployUrl": "/",
-            "baseHref": "/"
+            "deployUrl": "<stark-dummy-deploy-url>",
+            "baseHref": "<stark-dummy-base-href>"
```

Comment:
The changes in these 2 lines should be reverted. They are changed when you run `npm run build:showcase:ghpages` which I guess is the command you used to test :p

Reply:
Ooops... I didn't see them 😛